src/components/admin/Pagination.js

```diff
@@ -65,13 +65,12 @@ function Pagination({ pageCount, current, navigate }) {
         {
           (pageCount > 5 && current > 3) &&
           <li>
-            <a className="fr-pagination__link fr-displayed-lg">...</a>
+            <a className="fr-pagination__link fr-displayed-lg" href="/#" onClick={e => onClick(e, previousPage - 1)}>...</a>
```

Comment:
/# normal ?

Reply:
oui sinon le a est inaccessible

Reply:
c'est pour éviter une "redirection" vers l'accueil non ?
en tout cas le href dois être mis comme c'est une balise <a>

Reply:
oki, je crois que si in vire totalement le href, c'est plus cliquable (j'ai un doute du coup)

Reply:
Le bouton est grisé et non cliquable oui :)

Reply:
J'ai pas vu qu'il avait rendu invisible ma balise < a >